src/openrct2/rct2/S6Importer.cpp

```diff
@@ -429,7 +429,7 @@ extern "C"
         {
             s6Importer->LoadSavedGame(path);
         }
-        return s6Importer->HasInvalidObjects();
+        return !s6Importer->HasNoInvalidObjects();
```

Comment:
I'm not sure if it's a good idea to have the C function act differently from the C++ counterpart. Probably better to change it as well.